src/Uno.Extensions.Toolkit.UI/ThemeService.cs

```diff
@@ -21,35 +22,77 @@ internal class ThemeService : IThemeService
 		_dispatcher = dispatcher;
 		_logger = logger;
 
-		_ = _dispatcher.ExecuteAsync(async ct =>
+		Console.WriteLine($"Thread access: {_dispatcher.HasThreadAccess}");
```

Comment:
We shouldn't be writing this to the console. Let's either drop it or use the logger